tests/ga/test_exthandlers_download_extension.py

```diff
@@ -95,44 +98,52 @@ def _assert_download_and_expand_succeeded(self):
         self.assertTrue(os.path.exists(self._get_extension_package_file()), "The extension package was not downloaded to the expected location")
         self.assertTrue(os.path.exists(self._get_extension_command_file()), "The extension package was not expanded to the expected location")
 
+    @staticmethod
```

Comment:
The behaviours here used to be in implemented by WireProtocol.download_ext_handler_pkg, but now they are implemented by WireClient.stream